sentinelhub/sentinelhub_batch.py

```diff
@@ -247,14 +253,18 @@ def geometry(self):
         return Geometry(geometry, crs)
 
     @staticmethod
-    def iter_requests(user_id=None, config=None, **kwargs):
+    def iter_requests(user_id=None, search=None, sort=None, config=None, **kwargs):
         """ Iterate existing batch requests
 
         `Batch API reference
         <https://docs.sentinel-hub.com/api/latest/reference/#operation/getAllBathProcessRequests>`__
```

Comment:
This link no longer works

Reply:
Nice catch. I fixed it now and also checked all other links.